### ISSUE TITLE
Dropbox Sync 

### ISSUE THREAD

**Issue by kelvinkad** [laurent22/joplin#11239]
### Operating system

Android

### Joplin version

3.0.8

### Current behaviour

When I sync notes to dropbox the file names on dropbox are all random numbers, making it very difficult to identify my notes.

### Expected behaviour

The dropbox file names is exactly the file names (titles) of joplin app.

**Comment by dpoulton-github** [comment]
**TL:DR**
Dropbox is not to be used as a "web front end" for your Joplin note data. It is used only to facilitate the transfer of sync data between clients. Direct editing may cause data corruption or loss.

**Explanation**
The sync files transferred to Dropbox are for the Joplin clients to use to sync. The notes are not intended to edited **on** Dropbox. The "random numbers" are unique references Joplin uses to keep track of the note data. Directly editing means that the Joplin clients (which maintain records of what needs adding amending or deleting) will not be aware of any changes. Direct edits may never be transferred to clients and moving or deleting a file directly on Dropbox could cause serious problems for all your Joplin clients. Direct editing may cause data corruption or loss.
